src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
 }
+
```

Comment:
Consider adding reset styles and some general pages styles to a separate file to keep your main scss clean